packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -248,11 +248,11 @@ There are number of new environment variables, and you may use this table as a r
 | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
 | CELO_IMAGE                           | The Docker image used for the Validator and proxy containers                                                                         |
 | NETWORK_ID                           | The Celo Baklava network chain ID                                                                                                    |
-| CELO_VALIDATOR_GROUP_RG_ADDRESS         | The `ReleaseGold` contract address for the Validator Group                                                                                          |
-| CELO_VALIDATOR_RG_ADDRESS         | The `ReleaseGold` contract address for the Validator                                                                                          |
-| CELO_VALIDATOR_GROUP_SIGNER_ADDRESS        | The address of the Validator Group signer authorized by the Validator Group Account                                                              |
+| CELO_VALIDATOR_GROUP_ADDRESS         | The `ReleaseGold` contract address for the Validator Group                                                                                          |
+| CELO_VALIDATOR_ADDRESS         | The `ReleaseGold` contract address for the Validator                                                                                          |
+| CELO_VALIDATOR_GROUP_SIGNER_ADDRESS        | The address of the validator signer address authorized by the Validator Group Account                                                              |
```

Comment:
nit: consider `The validator (group) signer address authorized by the Validator Group Account.`